### PR TITLE
Add google test library

### PR DESCRIPTION
**Pull request by fjtrujy**
## Description
This PR should compile once the https://github.com/pspdev/psptoolchain-allegrex/pull/37 has been merged

### CODE REVIEW

File: googletest/PSPBUILD [hunk 1]
```diff
@@ -0,0 +1,43 @@
+pkgname=googletest
+pkgver=1.14.0
+pkgrel=1
+pkgdesc="GoogleTest - Google Testing and Mocking Framework"
+arch=('mips')
+url="google.github.io/googletest/"
+license=('BSD')
```
Comment (sharkwouter) [inline googletest/PSPBUILD:7]: I think we could be a bit more specific here and set it to BSD-3-Clause

**Comment by sharkwouter** [comment]
The resulting package looks good now, I just had the one remark.

File: googletest/PSPBUILD (hunk 1 above)
Comment (fjtrujy) [reply googletest/PSPBUILD]: Done!

**Comment by sharkwouter** [comment]
Thanks!

**Pull request merged**